privaterelay/settings.py

```diff
@@ -226,7 +233,7 @@
 
 DJANGO_STATSD_ENABLED = config("DJANGO_STATSD_ENABLED", False, cast=bool)
 STATSD_DEBUG = config("STATSD_DEBUG", False, cast=bool)
-STATSD_ENABLED = DJANGO_STATSD_ENABLED or STATSD_DEBUG
+STATSD_ENABLED: bool = DJANGO_STATSD_ENABLED or STATSD_DEBUG
 STATSD_HOST = config("DJANGO_STATSD_HOST", "127.0.0.1")
 STATSD_PORT = config("DJANGO_STATSD_PORT", "8125")
 STATSD_PREFIX = config("DJANGO_STATSD_PREFIX", "fx.private.relay")
```

Comment:
question (non-blocking): I can't figure out why we declare the type of `STATSD_ENABLED` but none of these other ones? Does the default value in the `config` call effectively declare the type for `mypy`? I can't tell what the pattern is of when we do/don't declare the type of a `settings` variable.